app/views/adopters/_adopter_form.html.erb

```diff
@@ -12,9 +12,9 @@
 
         <% if @adopter.adoption_app.pets_branch == "no_pets" && @adopter.adoption_app.prev_pets_type.blank? %>
           <span class="label label-warning">First Time Pet Owner</span>
-        <% elsif @adopter.adoption_app.prev_pets_type == "Cats" && @adopter.dog_or_cat == "Dog" %>
+        <% elsif (@adopter.adoption_app.prev_pets_type == "Cats" || @adopter.adoption_app.prev_pets_type == "None") && (@adopter.adoption_app.current_pets_type == "Cats" || @adopter.adoption_app.current_pets_type == "None") && @adopter.dog_or_cat == "Dog" %>
           <span class="label label-warning">First Time Dog Owner</span>
-        <% elsif @adopter.adoption_app.prev_pets_type == "Dogs" && @adopter.dog_or_cat == "Cat"%>
+        <% elsif (@adopter.adoption_app.prev_pets_type == "Dogs" || @adopter.adoption_app.prev_pets_type == "None") && (@adopter.adoption_app.current_pets_type == "Dogs" || @adopter.adoption_app.current_pets_type == "None") && @adopter.dog_or_cat == "Cat"%>
```

Comment:
As a later kind of thing: These long elseif's would be nice to refactor into methods in the Adopter model to make the view easier to read.